packages/app-metadata-controller/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@metamask/app-metadata-controller",
+  "version": "1.0.0",
```

Can we set the version to `0.0.0` until the first release?
```suggestion
  "version": "0.0.0",
```